modules/message-lane/src/instant_payments.rs

```diff
@@ -0,0 +1,75 @@
+// Copyright 2019-2020 Parity Technologies (UK) Ltd.
```

Comment:
I'm not sure if `instant_payments` is the best name for this since from what it looks like its main purpose is to pay relayers for message delivery

Reply:
Its main purpose is to pay relayers just after proof-of-delivery has been received (instantly).

Reply:
I think that until we have a distinction between instant and non-instant payments for relayers this should be called `relayer_payments` :)

Reply:
(1) it isn't only about payments to relayer (2) it's an implementation of trait with specific that payments are made instantly. So it should not be called `relayr_payments`